docs/_config.yml

```diff
@@ -17,18 +17,20 @@ remote_theme: pmarsceill/just-the-docs@v0.4.1
 
 # Aux links for the upper right navigation
 aux_links:
-  'Docs':
+  'Home':
     - 'https://docs.lakefs.io/'
   'Blog':
     - 'https://lakefs.io/blog/'
-  'Community':
-    - 'https://lakefs.io/community/'
+  'Guides':
+    - 'https://lakefs.io/guides/'  
+  'Slack':
+    - 'https://lakefs.io/slack/'
   'GitHub':
     - 'https://github.com/treeverse/lakeFS'
-
+    
 buttons:
-  'lakeFS Cloud':
-      - 'https://lakefs.io/cloud-registration/'
+  'Try without installing':
+    - 'https://demo.lakefs.io/'
```

Comment:
Hi @michalwosk - from what I understand [demo.lakefs.io](http://demo.lakefs.io/) is the old playground environment, the new one is not released to production yet, yet there is a plan to release it today and destroy the old one next week. which other button would you like to have there instead?

Reply:
Yes, we will redirect in any case. No worries.